app/dashboard/models.py

```diff
@@ -377,7 +377,19 @@ def value_in_usdt(self):
             return None
 
     @property
-    def token_value_in_usdt(self):
+    def value_in_usdt(self):
+        decimals = 10 ** 18
+        if self.token_name == 'USDT':
+            return float(self.value_in_token)
+        if self.token_name == 'DAI':
+            return float(self.value_in_token / 10 ** 18)
+        try:
+            return round(float(convert_amount(self.value_in_eth, 'ETH', 'USDT', self.web3_created)) / decimals, 2)
```

Comment:
how do you know that `self.token_name` == `ETH`?

Reply:
Good point. It was a copy-paste of the current implementation. I will change that to self.token_name.

Reply:
Done.